docs/operations/karpenter.md

```diff
@@ -0,0 +1,77 @@
+
+## Installing
+
+Enable the Karpenter feature flag:
+
+```sh
+export KOPS_FEATURE_FLAGS="Karpenter"
+```
+
+Karpenter requires that external permissions for ServiceAccounts be enabled for the cluster. See [AWS IAM roles for ServiceAccounts documentation](/cluster_spec#service-account-issuer-discovery-and-aws-iam-roles-for-service-accounts-irsa) for how to enable this. 
+
+### Existing clusters
+
+On existing clusters, you can create a Karpenter InstanceGroup by adding the following to its InstanceGroup spec:
+
+```yaml
+spec:
+  manager: Karpenter
+```
+
+You also need to enable the Karpenter addon in the cluster spec:
+
+```yaml
+spec:
+  karpenter:
+    enabled: true
+```
+
+### New clusters
+
+On new clusters, you can simply add the `--instance-manager=karpenter` flag:
+
+```sh
+kops create cluster --name mycluster.example.com --cloud aws --networking=amazonvpc --zones=eu-central-1a,eu-central-1b --master-count=3 --yes --discovery-store=s3://discovery-store/
+```
+
+## Karpenter-managed InstanceGroups
+
+A Karpenter-managed InstanceGroup controls a corresponding Karpenter Provisioner resource. kOps will ensure that the Provisioner is configured with the correct AWS security groups, subnets, and launch templates. Just like with ASG-managed InstanceGroups, you can add labels and taints to Nodes and kOps will ensure those are added accordingly.
+
+Note that not all features of InstanceGroups are supported.
+
+## Instance Types
+
+If you do not specify a mixed instances policy, only the instance type specified by `spec.machineType` will be used. With Karpenter, one typically wants a wider range of instances to choose from. kOps supports both providing a list of instance types through `spec.mixedInstancesPolicy.instances` and providing instance type requirements through `spec.mixedInstancesPolicy.instanceRequirements`. See (/instance_groups)[InstanceGroup documentation] for more details.
+
+## Known limitations
+
+### Subnet selections
+
+kOps will ignore the InstanceGroup `spec.subnets` configuration and unconditionally add all subnets eligible to run Nodes to all Provisioners. It is not currently possible to create Provisioners limited only to certain subnets.
+
+### CNI expectations
+
+Karpenter assumes the CNI is `amazonaws` when it calculates the maximum number of Pods that fit on a node. For other CNIs this means that Karpenter will calculate too few Pods on for smaller instances and too many Pods on larger instances. The default Karpenter-managed InstanceGroup is configured to avoid very large instance types for this reason.
+
+### Karpenter-managed Launch Templates
+
+By default, Karpener will create its own launch templates for Provisioners. These launch templates will not work with kOps cluster for a number of reasons. Most importantly they do not use a supported AMI and they do not install and configure nodeup, the instance-side kOps component.
```

Comment:
```suggestion
On EKS, Karpener will create its own launch templates for Provisioners. These launch templates will not work with kOps cluster for a number of reasons. Most importantly they do not use a supported AMI and they do not install and configure nodeup, the instance-side kOps component. The Karpenter features that requires Karpenter to directly manage launch templates will not be available on kOps.
```

Maybe something like this instead?

Reply:
That's acceptable.

"requires" should be "require" in order to match number with the subject. Also consider "creates" instead of "will create".